common/lib/api-client/index.js

```diff
@@ -16,18 +16,14 @@ const {
 } = require('./middleware')
 const models = require('./models')
 
-let instance
-
-module.exports = function () {
-  if (instance) {
-    return instance
-  }
-
-  instance = new JsonApi({
+module.exports = function (req) {
+  const instance = new JsonApi({
```

Comment:
What's the reason for removing the singleton pattern here?

By removing this is means we'll be creating a new instance for every call. I'm trying to rack my brain but we had some import issues with creating a new instance each time.

I think it would be neater if we can retain the API client as a singleton but find a way to define the req with a particular call.